src/data/text/teachy_tv.h

```diff
@@ -88,26 +88,26 @@ const u8 gTeachyTvText_MatchupsScript1[] = _(
     "what I mean.\p"
     "All righty, here goes!\p"
     "Keep your eyes glued to the super\n"
-    "POKé DUDE SHOW!");
+    "Poké Dude Show!");
 const u8 gTeachyTvText_MatchupsScript2[] = _(
     "Is it possible to launch an attack\n"
     "that will inflict heavy damage?\p"
-    "Does the opposing POKéMON pose\n"
-    "a threat to your POKéMON?\p"
+    "Does the opposing Pokémon pose\n"
+    "a threat to your Pokémon?\p"
     "Is there any chance that it may\n"
     "have disastrously tough moves?\p"
     "Watch the type matchups to gain\n"
     "the upper hand!\p"
     "All righty, be seeing you!\p"
-    "Oh, for the COOL-type POKé DUDE,\n"
-    "AWESOME-type kids like you match\l"
+    "Oh, for the Cool-type Poké Dude,\n"
+    "Awesome-type kids like you match\l"
```

Comment:
the \l looks off, from everything I've seen so far, I wanna say it should be \$

Reply:
That's how it is in the vanilla branch. no changes shown so I'm not fixing this one.